### PR TITLE
Update session API docs with up-to-date examples

### PR DESCRIPTION
**Pull request by muodov**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -347,25 +428,11 @@ Create a new session
...
-| ip_follower | IP address of the follower  | 127.0.0.1 |
-| emails_send | list of email invitations sent | [{"to": [], "from": null}] |
-| queue_id |  | 0 |
-| pin | 4-digit PIN code that can be used to join the session | null |
-| queued |  | false |
-| meta | additional data attached to joining user | {} |
-| tags | array of session tags | ["tag1"] |
-
-
+| leader_link |  the link that must be opened in a ***leader*** (controlling) browser. It has a 2 minute time out attached to it, if not used within that time, the session will be closed. | https://surfly.com/mUBRJyslQStVXPEwUwFyQ |
```
Comment (jsnjack) [inline apiary.apib:430]: isn't it 30 sec?

File: apiary.apib [hunk 2]
```diff
@@ -304,32 +304,113 @@ A list of objects containing the following parameters:
 |  Name | Description | Example |
 | ---------------| ------------| --------|
 | session_id | unique ID of the session | fOBg4dXAPTHSoil6LywgfYA |
+| id | Always the same as session_id, left for backwards compatibility | fOBg4dXAPTHSoil6LywgfYA |
```
Comment (jsnjack) [inline apiary.apib]: there is no id in the old call format

File: apiary.apib [hunk 3]
```diff
@@ -304,32 +304,113 @@ A list of objects containing the following parameters:
...
-| viewer_link | the link that must be opened in a ***viewer*** browser | https://session.surfly.com/fHrD6TMmRrurMevgidxyw |
+| viewer_link | the link that must be opened in a ***viewer*** browser | https://surfly.com/123-456-789 |
 | start_time | time the session started | 2016-06-17T14:30:44.026111Z |
 | duration | length of the session (in seconds) | 123 |
-| queue_id |  | 3927 |
+| end_time | time the session finished | 2016-06-17T15:30:44.026111Z |
 | pin | 4-digit PIN code that can be used to join the session | 3927 |
-| queued |  | false |
-| start_url | page the session starts from | https://example.com/ |
+| queue_id | Always the same as pin, left for backwards compatibility | 3927 |
```
Comment (jsnjack) [inline apiary.apib]: What with the case of description?

File: apiary.apib (hunk 2 above)
Comment (muodov) [reply apiary.apib]: I assumed adding fields is ok, and we are using the same format for all those responses now

File: apiary.apib (hunk 1 above)
Comment (muodov) [reply apiary.apib:430]: Don't know actually, it is a copypaste from the current docs

File: apiary.apib (hunk 3 above)
Comment (muodov) [reply apiary.apib]: seriously

File: apiary.apib (hunk 1 above)
Comment (muodov) [reply apiary.apib:430]: I checked, it is 120 secs. Defined by `CONNECT_TIMEOUT` in session.py

File: apiary.apib (hunk 2 above)
Comment (jsnjack) [reply apiary.apib]: i think adding field is ok too, the question was mostly about a description. how is that left for backwards comparability if it is a new field? 😕

File: apiary.apib (hunk 3 above)
Comment (jsnjack) [reply apiary.apib]: :) seriously, why not just use existing styles instead of introducing your own?

File: apiary.apib [hunk 4]
```diff
@@ -304,32 +304,108 @@ A list of objects containing the following parameters:
 |  Name | Description | Example |
 | ---------------| ------------| --------|
 | session_id | unique ID of the session | fOBg4dXAPTHSoil6LywgfYA |
+| formatted_id | unique formatted ID of the session | 123-456-789 |
 | agent_id | ID of the agent | 2 |
-| viewer_link | the link that must be opened in a ***viewer*** browser | https://session.surfly.com/fHrD6TMmRrurMevgidxyw |
+| viewer_link | the link that must be opened in a ***viewer*** browser | https://surfly.com/123-456-789 |
 | start_time | time the session started | 2016-06-17T14:30:44.026111Z |
 | duration | length of the session (in seconds) | 123 |
-| queue_id |  | 3927 |
```
Comment (jsnjack) [inline apiary.apib:311]: I don't think we should remove it as some of our clients might still use it

Comment (muodov) [reply apiary.apib:311]: It is not removed in fact. I only removed it from the docs to prevent new users from using it.

Comment (jsnjack) [reply apiary.apib:311]: How does the dredd test passes then? :suspect:

Comment (muodov) [reply apiary.apib:311]: I replaced a url in test_general to make it pass. I'll change it back before merging

**Pull request merged**